### ISSUE TITLE
Dastaset Preprocess

### ISSUE THREAD

**Issue by zhangpeng2001** [Chasel-Tsui/mmrotate-dcfl#5]
Hello, I want to train the model on the DOTA-v1.0 and I would like to know whether the **Data Processing Pipeline** (train_pipeline and test_pipeline) contains preprocessing operations such as  cropping images.

**Comment by Chasel-Tsui** [comment]
Yes, you need to crop the large images provided by the DOTA-v1.0 to small images of 1024*1024 with an overlap of 200 pixels. I used the tools provided by mmrotate to accomplish this, you can try it [here](https://github.com/open-mmlab/mmrotate/blob/main/tools/data/dota/README.md)
